src/inputs/RadioGroupField.test.tsx

```diff
@@ -16,4 +16,37 @@ describe("RadioGroupField", () => {
     );
     click(r.favoriteCheese_a);
   });
+
+  it("should disable only first option", async () => {
+    const r = await render(
+      <RadioGroupField
+        label={"Favorite cheese"}
+        value="a"
+        onChange={() => {}}
+        options={[
+          { value: "a", label: "Asiago", disabled: true },
+          { value: "b", label: "Burratta" },
+        ]}
+      />,
+    );
+    const radioInput = r.container.querySelector(`[data-testid="favoriteCheese_a"]`)!;
```

Comment:
Suggestion: Given how our `render` DSL works, this can probably be just:

```
expect(r.favoriteCheese_a).toBeDisabled();
```

I think?

Reply:
I've tried this approach, but it returns an error:

```
TypeError: Cannot read property 'message' of undefined

    at new JestAssertionError (~/homebound/beam/node_modules/jest-chain/dist/chain.js:10:18)
    at Object.newMatcher [as toBeDisabled] (~/homebound/beam/node_modules/jest-chain/dist/chain.js:31:17)
    at Object.<anonymous> (~/homebound/beam/src/inputs/RadioGroupField.test.tsx:32:31)
```

Reply:
Ah yeah, np that this is merged, but I forgot the `()`, so:

```
expect(r.favoriteCheese_a()).toBeDisabled()
```